vic/drivers/shared_all/src/initialize_parameters.c

```diff
@@ -55,7 +55,7 @@ initialize_parameters()
     param.EMISS_H2O = 0.98;
 
     // Soil Constraints
-    param.SOIL_RARC = 100.0;
```

Comment:
can you explain this change?

Reply:
I've done some more comprehensive evaluations of VIC at the Ameriflux towers and found that 250 s/m is actually a more optimal value than 100.

Reply:
That's fine but this change should probably go in a separate PR.

Reply:
OK, I've broken out the `RARC_SOIL` change into a separate issue (#767) and will back out that change from this PR.  I'll make a new PR for the `RARC_SOIL` change.